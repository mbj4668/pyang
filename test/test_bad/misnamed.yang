module misname {
    yang-version 1.1;
    namespace urn:test:misname;
    prefix mn;
}
