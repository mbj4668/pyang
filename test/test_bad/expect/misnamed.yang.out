misnamed.yang:1: warning: WBAD_MODULE_NAME
